Assignment2/Assignment2.java

```diff
@@ -0,0 +1,242 @@
+/*
+ * Classname: Assigment2.java
+ */
+import java.util.*;
+
+interface ShapeInterface{
```

Comment:
This is now a single interface. There are going to be certain shapes for which perimeter is not applied or volume is not applicable. For example, Circle is also implementing this interface and will get the volume method which is not applicable to it. This will be incorrect.